### ISSUE TITLE
[1.21.1] Unable to start on the 1.21.1 Neoforge server

### ISSUE THREAD

**Issue by 1254qwer** [Minecraft-LightLand/Twilight-Delight#31]
Unable to start on the 1.21.1 Neoforge server

[main/ERROR] [minecraft/Main]: Failed to start the minecraft server
net.neoforged.fml.ModLoadingException: Loading errors encountered:
        - Twilight Flavors & Delight (twilightdelight) has failed to load correctly
          java.lang.BootstrapMethodError: java.lang.RuntimeException: Attempted to load class net/minecraft/client/player/AbstractClientPlayer for invalid dist DEDICATED_SERVER

## Version

- MC: 1.21.1
- Neoforge: 21.1.51
- twilightdelight: 3.0.0+2
- FarmersDelight: 1.2.4
- twilightforest: 4.6.2780
- L2 Library: 3.0.2+4

## Logs

```
[21:18:05.191] [main/INFO] [Launcher/MODLAUNCHER]: ModLauncher running: args [--launchTarget, forgeserver, --fml.neoForgeVersion, 21.1.51, --fml.fmlVersion, 4.0.24, --fml.mcVersion, 1.21.1, --fml.neoFormVersion, 20240808.144430, nogui]
[21:18:05.198] [main/INFO] [Launcher/MODLAUNCHER]: JVM identified as Azul Systems, Inc. OpenJDK 64-Bit Server VM 21.0.4+7-LTS
[21:18:05.201] [main/INFO] [Launcher/MODLAUNCHER]: ModLauncher 11.0.4+main.d2e20e43 starting: java version 21.0.4 by Azul Systems, Inc.; OS Windows 11 arch amd64 version 10.0
[21:18:05.529] [main/INFO] [loading.ImmediateWindowHandler/]: ImmediateWindowProvider not loading because launch target is forgeserver
[21:18:05.647] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.7 Source=union:/F:/mctest2/libraries/net/fabricmc/sponge-mixin/0.15.2+mixin.0.8.7/sponge-mixin-0.15.2+mixin.0.8.7.jar%2375!/ Service=ModLauncher Env=SERVER
[21:18:05.888] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "server-1.21.1-20240808.144430-srg.jar" [locator: production server provider +net.neoforged:neoforge:21.1.51:server]
[21:18:05.910] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "neoforge-21.1.51-universal.jar" [locator: PathBasedLocator[name=neoforge, paths=[libraries\net\neoforged\neoforge\21.1.51\neoforge-21.1.51-universal.jar]], reader: mod manifest]
[21:18:05.923] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "FarmersDelight-1.21-1.2.4.jar" [locator: {mods folder locator at F:\mctest2\mods}, reader: mod manifest]
[21:18:05.931] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "l2library-3.0.2+4.jar" [locator: {mods folder locator at F:\mctest2\mods}, reader: mod manifest]
[21:18:05.938] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "twilightdelight-3.0.0+2.jar" [locator: {mods folder locator at F:\mctest2\mods}, reader: mod manifest]
[21:18:05.957] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "twilightforest-1.21.1-4.6.2780-universal.jar" [locator: {mods folder locator at F:\mctest2\mods}, reader: mod manifest]
[21:18:06.128] [main/INFO] [loading.moddiscovery.locators.JarInJarDependencyLocator/]: Found 9 dependencies adding them to mods collection
[21:18:06.128] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found gamelibrary file "l2modularblocks-3.0.0+4.jar" [parent: l2library-3.0.2+4.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.129] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "l2itemselector-3.0.8.jar" [parent: l2library-3.0.2+4.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.129] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found library file "s.tf-asm-4.6.2780.jar" [parent: twilightforest-1.21.1-4.6.2780-universal.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.129] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "l2tabs-3.0.5+2.jar" [parent: l2library-3.0.2+4.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.129] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found gamelibrary file "l2serial-3.0.9+4.jar" [parent: l2library-3.0.2+4.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.129] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "l2menustacker-3.0.9.jar" [parent: l2library-3.0.2+4.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.130] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found gamelibrary file "Registrate-MC1.21-1.3.0+55.jar" [parent: l2library-3.0.2+4.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.131] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found gamelibrary file "mixinextras-neoforge-0.4.1.jar" [parent: neoforge-21.1.51-universal.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.131] [main/INFO] [loading.moddiscovery.ModDiscoverer/SCAN]: Found mod file "l2core-3.0.7+42.jar" [parent: l2library-3.0.2+4.jar, locator: jarinjar, reader: mod manifest]
[21:18:06.481] [main/INFO] [loading.FMLServiceProvider/CORE]: Loading coremod script-engine for [neoforge]
[21:18:08.024] [main/INFO] [mixin/]: Compatibility level set to JAVA_21
[21:18:08.044] [main/INFO] [LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeserver' with arguments [nogui]
WARN StatusConsoleListener Advanced terminal features are not available in this environment
[21:18:10] [main/INFO] [MixinExtras|Service/]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.4.1).
[21:18:19] [modloading-worker-0/INFO] [tw.be.TFBeanContext/]: Starting Bean Context
[21:18:19] [modloading-worker-0/INFO] [ne.ne.ne.co.NeoForgeMod/NEOFORGE-MOD]: NeoForge mod loading, version 21.1.51, for MC 1.21.1
[21:18:19] [modloading-worker-0/INFO] [tw.be.TFBeanContext/]: Bean Context loaded in 161 ms
[21:18:20] [modloading-worker-0/ERROR] [ne.ne.fm.co.as.RuntimeDistCleaner/DISTXFORM]: Attempted to load class net/minecraft/client/player/AbstractClientPlayer for invalid dist DEDICATED_SERVER
[21:18:20] [modloading-worker-0/ERROR] [ne.ne.fm.ja.FMLModContainer/LOADING]: Failed to create mod instance. ModID: twilightdelight, class dev.xkmc.twilightdelight.init.TwilightDelight
java.lang.BootstrapMethodError: java.lang.RuntimeException: Attempted to load class net/minecraft/client/player/AbstractClientPlayer for invalid dist DEDICATED_SERVER
        at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.registrate.TDEffects.<clinit>(TDEffects.java:18) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
        at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.registrate.delight.DelightFood.<clinit>(DelightFood.java:21) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
        at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.TwilightDelight.<init>(TwilightDelight.java:65) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
        at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?] {}
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?] {}
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:115) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.lambda$constructMods$3(ModLoader.java:153) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.lambda$dispatchParallelTask$7(ModLoader.java:231) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934) [?:?] {}
        at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911) [?:?] {}
        at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483) [?:?] {}
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) [?:?] {}
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) [?:?] {}
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) [?:?] {}
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) [?:?] {}
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) [?:?] {}
Caused by: java.lang.RuntimeException: Attempted to load class net/minecraft/client/player/AbstractClientPlayer for invalid dist DEDICATED_SERVER
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.common.asm.RuntimeDistCleaner.processClassWithFlags(RuntimeDistCleaner.java:60) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:94) ~[modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:57) ~[modlauncher-11.0.4.jar%2362!/:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:190) ~[securejarhandler-3.0.8.jar:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.lambda$findClass$20(ModuleClassLoader.java:298) ~[securejarhandler-3.0.8.jar:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:311) ~[securejarhandler-3.0.8.jar:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:298) ~[securejarhandler-3.0.8.jar:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:214) ~[securejarhandler-3.0.8.jar:?] {}
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?] {}
        ... 17 more
[21:18:20] [main/FATAL] [ne.ne.fm.ModLoader/LOADING]: Failed to wait for future Mod Construction, 1 errors found
[21:18:20] [main/ERROR] [minecraft/CrashReport]: Negative index in crash report handler (0/25)
[21:18:20] [main/FATAL] [ne.ne.ne.co.NeoForgeMod/]: Preparing crash report with UUID d4e770c4-30cf-481d-b318-5252ac021183
[21:18:20] [main/FATAL] [ne.ne.ne.se.lo.ServerModLoader/]: Crash report saved to .\crash-reports\crash-2024-09-14_21.18.20-fml.txt
[21:18:20] [main/FATAL] [ne.ne.ne.co.NeoForgeMod/]: Preparing crash report with UUID d3300f2c-8aeb-480d-beb7-69c6336d0a7d
---- Minecraft Crash Report ----
// I let you down. Sorry :(

Time: 2024-09-14 21:18:20
Description: Mod loading failures have occurred; consult the issue messages for more details

net.neoforged.neoforge.logging.CrashReportExtender$ModLoadingCrashException: Mod loading has failed


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: main
Stacktrace:
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.common.asm.RuntimeDistCleaner.processClassWithFlags(RuntimeDistCleaner.java:60) ~[loader-4.0.24.jar%2360!/:4.0] {}
-- Mod loading issue for: twilightdelight --
Details:
        Caused by 0: java.lang.BootstrapMethodError: java.lang.RuntimeException: Attempted to load class net/minecraft/client/player/AbstractClientPlayer for invalid dist DEDICATED_SERVER
                at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.registrate.TDEffects.<clinit>(TDEffects.java:18) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
                at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.registrate.delight.DelightFood.<clinit>(DelightFood.java:21) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
                at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.TwilightDelight.<init>(TwilightDelight.java:65) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
                at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?] {}
                at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?] {}
                at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?] {}
                at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:115) ~[loader-4.0.24.jar%2360!/:4.0] {}
                at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.lambda$constructMods$3(ModLoader.java:153) ~[loader-4.0.24.jar%2360!/:4.0] {}
                at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.lambda$dispatchParallelTask$7(ModLoader.java:231) ~[loader-4.0.24.jar%2360!/:4.0] {}
                at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934) ~[?:?] {}
                at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911) ~[?:?] {}
                at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483) ~[?:?] {}
                at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) ~[?:?] {}
                at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) ~[?:?] {}
                at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) ~[?:?] {}
                at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) ~[?:?] {}
                at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) ~[?:?] {}

        Mod file: /F:/mctest2/mods/twilightdelight-3.0.0+2.jar
        Failure message: Twilight Flavors & Delight (twilightdelight) has failed to load correctly
                java.lang.BootstrapMethodError: java.lang.RuntimeException: Attempted to load class net/minecraft/client/player/AbstractClientPlayer for invalid dist DEDICATED_SERVER
        Mod version: 3.0.0+2
        Mod issues URL: <No issues URL found>
        Exception message: java.lang.RuntimeException: Attempted to load class net/minecraft/client/player/AbstractClientPlayer for invalid dist DEDICATED_SERVER
Stacktrace:
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.common.asm.RuntimeDistCleaner.processClassWithFlags(RuntimeDistCleaner.java:60) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:94) ~[modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:57) ~[modlauncher-11.0.4.jar%2362!/:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:190) ~[securejarhandler-3.0.8.jar:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.lambda$findClass$20(ModuleClassLoader.java:298) ~[securejarhandler-3.0.8.jar:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:311) ~[securejarhandler-3.0.8.jar:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:298) ~[securejarhandler-3.0.8.jar:?] {}
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:214) ~[securejarhandler-3.0.8.jar:?] {}
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?] {}
        at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.registrate.TDEffects.<clinit>(TDEffects.java:18) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
        at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.registrate.delight.DelightFood.<clinit>(DelightFood.java:21) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
        at TRANSFORMER/twilightdelight@3.0.0+2/dev.xkmc.twilightdelight.init.TwilightDelight.<init>(TwilightDelight.java:65) ~[twilightdelight-3.0.0+2.jar%23127!/:3.0.0+2] {re:classloading}
        at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?] {}
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?] {}
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:115) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.lambda$constructMods$3(ModLoader.java:153) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.lambda$dispatchParallelTask$7(ModLoader.java:231) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934) ~[?:?] {}
        at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911) ~[?:?] {}
        at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483) ~[?:?] {}
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) ~[?:?] {}
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) ~[?:?] {}
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) ~[?:?] {}
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) ~[?:?] {}
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) ~[?:?] {}


-- System Details --
Details:
        Minecraft Version: 1.21.1
        Minecraft Version ID: 1.21.1
        Operating System: Windows 11 (amd64) version 10.0
        Java Version: 21.0.4, Azul Systems, Inc.
        Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Azul Systems, Inc.
        Memory: 84874000 bytes (80 MiB) / 387973120 bytes (370 MiB) up to 4257218560 bytes (4060 MiB)
        CPUs: 16
        Processor Vendor: AuthenticAMD
        Processor Name: AMD Ryzen 7 5800H with Radeon Graphics
        Identifier: AuthenticAMD Family 25 Model 80 Stepping 0
        Microarchitecture: Zen 3
        Frequency (GHz): 3.19
        Number of physical packages: 1
        Number of physical CPUs: 8
        Number of logical CPUs: 16
        Graphics card #0 name: NVIDIA GeForce RTX 3060 Laptop GPU
        Graphics card #0 vendor: NVIDIA
        Graphics card #0 VRAM (MiB): 6144.00
        Graphics card #0 deviceId: VideoController1
        Graphics card #0 versionInfo: 32.0.15.5612
        Memory slot #0 capacity (MiB): 8192.00
        Memory slot #0 clockSpeed (GHz): 3.20
        Memory slot #0 type: DDR4
        Memory slot #1 capacity (MiB): 8192.00
        Memory slot #1 clockSpeed (GHz): 3.20
        Memory slot #1 type: DDR4
        Virtual memory max (MiB): 43273.91
        Virtual memory used (MiB): 37770.46
        Swap memory total (MiB): 27041.66
        Swap memory used (MiB): 5767.71
        Space in storage for jna.tmpdir (MiB): <path not set>
        Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
        Space in storage for io.netty.native.workdir (MiB): <path not set>
        Space in storage for java.io.tmpdir (MiB): available: 23367.15, total: 465950.00
        Space in storage for workdir (MiB): available: 31561.05, total: 849808.00
        JVM Flags: 0 total;
        ModLauncher: 11.0.4+main.d2e20e43
        ModLauncher launch target: forgeserver
        ModLauncher services:
                sponge-mixin-0.15.2+mixin.0.8.7.jar mixin PLUGINSERVICE
                loader-4.0.24.jar slf4jfixer PLUGINSERVICE
                loader-4.0.24.jar runtime_enum_extender PLUGINSERVICE
                at-modlauncher-10.0.1.jar accesstransformer PLUGINSERVICE
                loader-4.0.24.jar runtimedistcleaner PLUGINSERVICE
                modlauncher-11.0.4.jar mixin TRANSFORMATIONSERVICE
                modlauncher-11.0.4.jar fml TRANSFORMATIONSERVICE
        FML Language Providers:
                javafml@4.0
                lowcodefml@4.0
                minecraft@4.0
        Mod List:
                FarmersDelight-1.21-1.2.4.jar                     |Farmer's Delight              |farmersdelight                |1.2.4               |Manifest: NOSIGNATURE
                server-1.21.1-20240808.144430-srg.jar             |Minecraft                     |minecraft                     |1.21.1              |Manifest: NOSIGNATURE
                l2tabs-3.0.5+2.jar                                |L2Tabs                        |l2tabs                        |3.0.5+2             |Manifest: NOSIGNATURE
                l2itemselector-3.0.8.jar                          |L2ItemSelector                |l2itemselector                |3.0.8               |Manifest: NOSIGNATURE
                l2library-3.0.2+4.jar                             |L2Library                     |l2library                     |3.0.2+4             |Manifest: NOSIGNATURE
                l2menustacker-3.0.9.jar                           |L2MenuStacker                 |l2menustacker                 |3.0.9               |Manifest: NOSIGNATURE
                twilightforest-1.21.1-4.6.2780-universal.jar      |The Twilight Forest           |twilightforest                |4.6.2780            |Manifest: NOSIGNATURE
                twilightdelight-3.0.0+2.jar                       |Twilight Flavors & Delight    |twilightdelight               |3.0.0+2             |Manifest: NOSIGNATURE
                neoforge-21.1.51-universal.jar                    |NeoForge                      |neoforge                      |21.1.51             |Manifest: NOSIGNATURE
                l2core-3.0.7+42.jar                               |L2Core                        |l2core                        |3.0.7+42            |Manifest: NOSIGNATURE
        Crash Report UUID: d3300f2c-8aeb-480d-beb7-69c6336d0a7d
        FML: 4.0.24
        NeoForge: 21.1.51
[21:18:20] [main/ERROR] [minecraft/Main]: Failed to start the minecraft server
net.neoforged.fml.ModLoadingException: Loading errors encountered:
        - Twilight Flavors & Delight (twilightdelight) has failed to load correctly
          java.lang.BootstrapMethodError: java.lang.RuntimeException: Attempted to load class net/minecraft/client/player/AbstractClientPlayer for invalid dist DEDICATED_SERVER

        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.waitForFuture(ModLoader.java:281) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.dispatchParallelTask(ModLoader.java:244) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.constructMods(ModLoader.java:152) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.ModLoader.gatherAndInitializeMods(ModLoader.java:142) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at TRANSFORMER/neoforge@21.1.51/net.neoforged.neoforge.internal.CommonModLoader.begin(CommonModLoader.java:49) ~[neoforge-21.1.51-universal.jar%23124!/:?] {re:classloading}
        at TRANSFORMER/neoforge@21.1.51/net.neoforged.neoforge.server.loading.ServerModLoader.load(ServerModLoader.java:34) ~[neoforge-21.1.51-universal.jar%23124!/:?] {re:classloading}
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.Main.main(Main.java:124) ~[server-1.21.1-20240808.144430-srg.jar%23123!/:?] {re:classloading}
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?] {}
        at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[?:?] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:136) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:128) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.loading.targets.CommonServerLaunchHandler.runService(CommonServerLaunchHandler.java:32) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/fml_loader@4.0.24/net.neoforged.fml.loading.targets.CommonLaunchHandler.lambda$launchService$4(CommonLaunchHandler.java:118) ~[loader-4.0.24.jar%2360!/:4.0] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.Launcher.run(Launcher.java:103) [modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.Launcher.main(Launcher.java:74) [modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-11.0.4.jar%2362!/:?] {}
        at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-11.0.4.jar%2362!/:?] {}
        at cpw.mods.bootstraplauncher@2.0.2/cpw.mods.bootstraplauncher.BootstrapLauncher.run(BootstrapLauncher.java:210) [bootstraplauncher-2.0.2.jar:?] {}
        at cpw.mods.bootstraplauncher@2.0.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:69) [bootstraplauncher-2.0.2.jar:?] {}
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?] {}
        at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[?:?] {}
        at net.neoforged.serverstarterjar.Main.mainLaunch(Main.java:266) [server.jar:0.1.25] {}
        at net.neoforged.serverstarterjar.Main.main(Main.java:70) [server.jar:0.1.25] {}
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?] {}
        at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[?:?] {}
        at net.neoforged.serverstarterjar.Main8.main(Main8.java:19) [server.jar:0.1.25] {}
```

**Comment by lcy0x1** [comment]
Will fix ASAP

**Comment by lcy0x1** [comment]
Fixed in 3.0.0+3
